src/app/data-client.spec.ts

```diff
@@ -1716,3 +1723,93 @@ describe('DataPipelineClient tests', () => {
     });
   });
 });
+
+describe('fileUpload tests', () => {
+  const partId = 'testPartId';
+  const binaryData = new Uint8Array([1, 2, 3, 4, 5]);
```

Comment:
can you please test with data that is large enough to actually validate that your chunking logic works correctly?